### ISSUE TITLE
Documentation on how to mark the unlabeled data? 

### ISSUE THREAD

**Issue by rbaehr** [pulearn/pulearn#8]
When setting up X and y arrays, should the unlabeled data be marked with y = -1? And the positive data should be marked with y = 1?

**Comment by shaypal5** [comment]
Hey Ross,

It actually varies between implementations; this is a by-product of this package simply being a collection of existing implementations.

The Elkanoto classifiers (`ElkanotoPuClassifier` and `WeightedElkanotoPuClassifier`) indeed assume `y=1` for positively labeled data and `y=-1` for unknown labels.

On the other hand, `BaggingPuClassifier` treats any `y<1` as an unknown label.

So using -1 as a negative label should work for both (let me know if it does not), but they still don't behave exactly the same.

**Comment by shaypal5** [comment]
I'm leaving this open as a task to add the proper documentation.

Thank you for pointing this out! It sourly missing! :)

**Comment by alencn1024** [comment]
Hello, kindly to ask if there is somebody working on this issue, if not, I'm glad to undertake it and I'm a complete beginner for  this. @shaypal5

**Comment by shaypal5** [comment]
Yes, nobody is working on it! Please feel free to contribute!
@alencn1024

**Comment by asitstands** [comment]
@shaypal5 Hi, I'm trying to use `ElkanotoPuClassifier` with [scikit-learn interface of xgboost](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBClassifier). It fails with an error saying that labels should be 0 or 1. Marking unlabeled as `-1` does not work with xgboost. Any workaround is there?

**Comment by shaypal5** [comment]
Closed by #19 

(@asitstands will be opened as a separate PR)
